airflow/gcp/operators/bigquery.py

```diff
@@ -242,9 +245,14 @@ class BigQueryGetDataOperator(BaseOperator):
     :type dataset_id: str
     :param table_id: The table ID of the requested table. (templated)
     :type table_id: str
+    :param start_index: (Optional) Start row index of the table.
+    :type start_index: str
     :param max_results: The maximum number of records (rows) to be fetched
         from the table. (templated)
-    :type max_results: str
+    :type max_results: int
```

Comment:
If we change the type of argument we need to mention that in the UPDATING.md so that people will see that before they run into an error.

Reply:
Didn't know that, thanks for clarifying. Will update it now.

Reply:
This is only required if the change is not backward compatible. In this case, is backward compatible.